### ISSUE TITLE
validate问题

### ISSUE THREAD

**Issue by 363325971** [SoftwareGift/FeatherNets_Face-Anti-spoofing-Attack-Detection-Challenge-CVPR2019#24]
你好，我在训练完一个epoch后进行validata后出错，经查是在
         _,predicted = torch.max(soft_output.data, 1)
         predicted = predicted.to('cpu').detach().numpy()

这里的predicted不是包含0,1的tensor，是一个这样的tensor
        Tensor: tensor([920, 744, 920, 818, 416, 920, 818, 920, 919, 920, 818, 818, 920, 974,
        920, 855, 919, 744, 503, 818, 744, 920, 818, 684, 818, 503, 920, 919,
        920, 818, 919, 974], device='cuda:0')

导致在后面计算confusion_matrix的时候无法正确计算tn, fp, fn, tp，
请问是哪里有问题吗？ 我是从train_list里随机取的2000个样本做val_list的，其他地方没有改动

**Comment by aloha000** [comment]
你好，请问你是用什么数据集跑的？原数据集我下载不了，求分享

**Comment by 363325971** [comment]
我用的就是百度云上的那个数据

**Comment by aloha000** [comment]
百度云的链接好像已经没有了，可以发我一下吗？谢谢～

**Comment by 363325971** [comment]
Duplicate of #2 这里有啊

**Comment by aloha000** [comment]
不知道为什么我这里readme没有显示 多谢～～

**Comment by aloha000** [comment]
请问是应该把PAD文件夹放在data目录里，还是把PAD目录下的文件放在data里？可以加您微信聊一下吗？小白做起来有点费劲～

**Comment by jiagh2010** [comment]
@aloha000   怎么做的？？数据

**Comment by chenkingwen** [comment]
我是用自己的数据集跑的，在val的某个阶段出问题的，请问怎么解决？
